commit aebae9835f39d7925dfc71819bb31e8445e12a8f
Author: Lukas Rusak <lorusak@gmail.com>
Date:   Thu May 31 23:44:03 2018 -0700

    meson: only build vl_winsys_dri.c when x11 platform is used

diff --git a/src/gallium/auxiliary/meson.build b/src/gallium/auxiliary/meson.build
index 584cbe4509..857001e12c 100644
--- a/src/gallium/auxiliary/meson.build
+++ b/src/gallium/auxiliary/meson.build
@@ -453,7 +453,7 @@ files_libgalliumvl = files(
 )
 
 files_libgalliumvlwinsys = files('vl/vl_winsys.h')
-if with_dri2
+if with_dri2 and with_platform_x11
   files_libgalliumvlwinsys += files('vl/vl_winsys_dri.c')
   if with_dri3
     files_libgalliumvlwinsys += files('vl/vl_winsys_dri3.c')
